### ISSUE TITLE
Atom JavaScript snippets - regression

### ISSUE THREAD

**Issue by Alexey-T** [OlehL/cuda_snippets#30]
package "Atom JavaScript" snippets.
These don't work now. Console error "wrong snippet". @OlehL 

```
name=Anonymous Function
id=f
lex=JavaScript
text=
function (${1}) {
	${2}
}
```
--
```
name=Anonymous Generator
id=g
lex=JavaScript
text=
function* (${1}) {
	${2}
}
```

**Comment by OlehL** [comment]
the same as #31 
fixed in 11b41a7b6eba330e35e3b3d5ca877624900cb6d4
pls test

**Comment by Alexey-T** [comment]
Will talk in #31 . closing.
